spec/type.spec.js

```diff
@@ -0,0 +1,150 @@
+goog.require('plain');
```

Comment:
This is missing from the previous test file - why is it now necessary?

Reply:
Karma links all of the files together, but we don't include the modules in the library by default. If we did, there would be some errors about requiring plain multiple times, so it's best to leave it in the spec file.Previously, all the files were required with some html.

Reply:
Thanks for the info! LGTM.